doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -161,6 +161,154 @@ This is a reference configuration that can be modified in the production firmwar
+       Three group keys are assumed for each Matter fabric.
+
+The default slots range allows storing more cryptographic materials than the minimum required according to the Matter specification.
+A minimum of 46 slots is required for cryptographic materials, including the DAC private key to support 5 fabrics (minimum required by the specification).
+A single Matter fabric requires at least nine KMU slots.
+
+The default range has been chosen to support up to eight Matter fabrics, 32 ICD keys (16 ICD users), and 24 group keys.
+You can change the default Kconfig option for each crypto material to adjust the number of slots, but you must ensure that the total number of slots fits within the defined range.
+If you overlap the slots range, the appropriate compilation error will be shown:
+
+.. code-block:: console
+
+    error: "The number of slots exceeds the range of the KMU defined in CONFIG_CHIP_KMU_SLOT_RANGE_START and CONFIG_CHIP_KMU_SLOT_RANGE_END"
+
+For example, to fill the default slots range and support 8 Matter fabrics, 3 group keys per fabric, and 2 ICD users per fabric, set the :kconfig:option:`CONFIG_CHIP_MAX_FABRICS` Kconfig option to 8.
+
+Slots arrangement within the slot range defined by :kconfig:option:`CONFIG_CHIP_KMU_SLOT_RANGE_START` and :kconfig:option:`CONFIG_CHIP_KMU_SLOT_RANGE_END` Kconfig options is calculated automatically in the code in the following way:
+
+.. list-table:: KMU slots arrangement for Matter crypto materials
+  :widths: auto
+  :header-rows: 1
+
+  * - Crypto material
+    - Slot range start
+    - Slot range end
+    - Default slot range for 8 Matter fabrics to fit the default range (100-180)
+  * - Node Operational Certificate (NOC) private key
+    - :kconfig:option:`CONFIG_CHIP_KMU_SLOT_RANGE_START`
+    - :kconfig:option:`CONFIG_CHIP_KMU_SLOT_RANGE_START` + (:kconfig:option:`CONFIG_CHIP_MAX_FABRICS` * 2)
+    - 100 - 115
+  * - Intermittently Connected Device (ICD) Token + ICD symmetric Key
+    - End of NOC slots
+    - End of NOC slots + (:kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` * :kconfig:option:`CONFIG_CHIP_MAX_FABRICS` * 2)
+    - 116 - 147
+  * - Group keys
+    - End of ICD slots
+    - End of ICD slots + (3 * :kconfig:option:`CONFIG_CHIP_MAX_FABRICS`)
+    - 148 - 171
+  * - Device Attestation Certificate (DAC) private key
+    - Configurable with :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU_SLOT_ID`
+    - DAC private key start slot + 1 (non-encrypted) or + 3 (encrypted)
+    - 176 - 179 (encrypted) or 178 - 179 (non-encrypted)
+
+.. important::
+
+  Once a slot range is defined, it should not be changed during the firmware update process.
+  Changing the slot range will result in the loss of all cryptographic materials stored in the KMU.
+  To extend the slot range beyond the previously defined range, you must complete one of the following actions, depending on whether you want to keep or remove the data:
+
+  .. tabs::
+
+    .. tab:: Remove the data
```

Comment:
To be honest, i would just leave the part and instruction how to keep the data. Otherwise, ofc, re-flashing is necessary.